src/configuration.cc

```diff
@@ -156,6 +162,25 @@ int Configuration::ParseArguments(int ac, char** av) {
                 << std::endl;
       return -1;
     }
+    if (flags.count("options")) {
```

Comment:
We should probably do this after the `ParseConfigFile` call, to allow explicit config option settings to override the configuration file.

Reply:
Done. Also updated ParseConfiguration to actually pick up only the updated values.